app/Http/Controllers/Admin/NodesController.php

```diff
@@ -140,14 +139,12 @@ public function allocationRemoveBlock(Request $request, int $node): RedirectResp
     /**
      * Sets an alias for a specific allocation on a node.
      *
-     * @throws \Pterodactyl\Exceptions\Model\DataValidationException
-     * @throws \Pterodactyl\Exceptions\Repository\RecordNotFoundException
      */
     public function allocationSetAlias(AllocationAliasFormRequest $request): \Symfony\Component\HttpFoundation\Response
     {
-        $this->allocationRepository->update($request->input('allocation_id'), [
-            'ip_alias' => (empty($request->input('alias'))) ? null : $request->input('alias'),
-        ]);
+        $allocation = Allocation::query()->findOrFail($request->input('allocation_id'));
```

Comment:
Is there a reason to select the allocation just to end up updating it?  Wouldn't it be simpler to do a single update?